src/Powercord/plugins/pc-spotify/Modal/contextMenuGroups.js

```diff
@@ -187,5 +187,20 @@ module.exports = (state, onButtonClick, hasCustomAuth, hasControlsHidden, hasIco
     icon: hasIconsHidden ? '' : 'copy-duotone',
     onClick: () =>
       clipboard.writeText(state.currentItem.url)
+  }, {
+    type: 'button',
+    name: 'Send album URL to Channel',
```

Comment:
"Album", should be uppercased for consistency
Should be above the 1st Copy URL for consistency (Group all sends, group all copys)